mock/dbTransactionsHandlerStub.go

```diff
@@ -0,0 +1,33 @@
+package mock
+
+import (
+	"bytes"
+
+	"github.com/ElrondNetwork/elastic-indexer-go/data"
+	nodeData "github.com/ElrondNetwork/elrond-go/data"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/data/indexer"
+)
+
+type DBTransactionProcessorStub struct {
+}
+
+func (tps *DBTransactionProcessorStub) PrepareTransactionsForDatabase(_ *block.Body, _ nodeData.HeaderHandler, _ *indexer.Pool) *data.PreparedResults {
+	panic("implement me")
```

Comment:
All functions from this struct cause panics. Do we really use this stub? Missing comments on the exported items. Please fix this file.

Reply:
done.